testdata/hello.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
Should there be a test where the file name also contains utf-8 outside of the ASCII set? Conceivably in a different PR at some point? I'd expect that something would break. (Anecdotally, there is some kind of issue with some file names that I'll file a bug for.)

Reply:
I see the issue. It's a general issue with setting tag, unrelated to dump files. I'll have a fix soon.

Reply:
I've created PR https://github.com/rjkroege/edwood/pull/269 with the test.